.github/workflows/nightly.yaml

```diff
@@ -92,31 +92,34 @@ jobs:
           command: udeps
           args: --all-targets
 
-  test-docker:
-    # We only test the build in `test-all`; this also runs tests.
-    runs-on: ubuntu-latest
-    steps:
-      - name: 📂 Checkout code
-        uses: actions/checkout@v3
-
-      - uses: docker/setup-buildx-action@v2
-
-      - name: Build
-        uses: docker/build-push-action@v4
-        with:
-          tags: prql:latest
-          # Use the GHA cache
-          load: true
-          cache-from: type=gha
-          cache-to: type=gha,mode=max
-
-      # https://aschmelyun.com/blog/using-docker-run-inside-of-github-actions/
-      - name: Test
-        uses: addnab/docker-run-action@v3
-        with:
-          image: prql:latest
-          options: -v ${{ github.workspace }}/:/src
-          run: task test-rust
+  # We now use the devcontainer. TODO: is it possible to have a similar test for
+  # that? Or that would require VSCode to install the dependencies?
```

Comment:
@eitsupi would you have any thoughts here?

No great stress. But I generally find that if we don't test something, it breaks, and it breaks further away from the cause!

Reply:
I think we can use the `docker run` command as before or use the devcontainer CLI to run JS and Python tasks in addition to Rust.
However, these take time to execute, so we need to think carefully about which ones to test.

Reply:
What tests could we run?

Because we build the base image in CI, but other commands — like `task` — are only built when builds the devcontainer in VSCode.

Ideally we would run `task test-rust`, like we did in this test.

Reply:
I don't think we need to worry much about runtime, since this only runs once a day